src/ralph/admin/filters.py

```diff
@@ -535,6 +535,32 @@ def queryset(self, request, queryset):
         return queryset
 
 
+class BaseObjectHostnameFilter(SimpleListFilter):
+    title = _('Hostname')
+    parameter_name = 'hostname'
+    template = 'admin/filters/text_filter.html'
+
+    def queryset(self, request, queryset):
+        if not self.value():
+            return queryset
+        queries = [
+            Q(hostname__icontains=self.value()) |
+            Q(ethernet_set__ipaddress__hostname__icontains=self.value())
```

Comment:
could you add this condition (for hostname of ip) to DCHostHostnameFilter https://github.com/allegro/ralph/blob/ng/src/ralph/data_center/admin.py#L79 ?